src/extensions/cookie/bootstrap-table-cookie.js

```diff
@@ -193,7 +192,8 @@
                 });
 
                 bootstrapTable.initColumnSearch(cachedFilters);
-                bootstrapTable.refresh();
+                bootstrapTable.options.filterControlValuesLoaded = true;
+                bootstrapTable.initServer();
```

Comment:
I've kept my implementation which calls the internal `initServer` instead of the public `refresh` (which triggers a 'refresh' event). The reason is that this part of the code is simply a "delayed init" and not really a refresh. At the init stage in which this code is executed the bootstrap table would normally execute `initServer` without this cookie plugin.

Note that the user experience could be improved by cutting down the length of the init delay from 250ms (see below) to a much smaller number but I don't know why it is such a large number. (Perhaps to avoid interference with other plugins that are also using delays for their init?)